lib/generators/manageiq/plugin/templates/bin/setup

```diff
@@ -2,8 +2,14 @@
 require 'pathname'
 
 gem_root = Pathname.new(__dir__).join("..")
+spec_manageiq = gem_root.join("spec/manageiq")
 
-unless gem_root.join("spec/manageiq").exist?
+if ENV.key?("MANAGEIQ_REPO")
+  puts "== Symlinking spec/manageiq to #{ENV["MANAGEIQ_REPO"]}"
+
+  spec_manageiq.rmtree if spec_manageiq.exist?
```

Comment:
Remove the old symlink/dir otherwise we'll end up with `spec/manageiq/manageiq -> ~/src/manageiq` :)